packages/paste-website/src/pages/components/separator/index.mdx

```diff
@@ -0,0 +1,151 @@
+---
+title: Separator - Components
+description: The Separator component can be used to create visual and semantic separation between two or more UI controls.
+---
+
+import {graphql} from 'gatsby';
+import {Separator} from '@twilio-paste/separator';
+import {Button} from '@twilio-paste/button';
+import {Text} from '@twilio-paste/text';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {Anchor} from '@twilio-paste/anchor';
+import {Box} from '@twilio-paste/box';
+import Changelog from '@twilio-paste/separator/CHANGELOG.md';
+import {SidebarCategoryRoutes} from '../../../constants';
+import {Callout, CalloutTitle, CalloutText} from '../../../components/callout';
+import {DoDont, Do, Dont} from '../../../components/DoDont';
+import {Table, Thead, Tbody, Tr, Td, Th} from '../../../components/table';
+import {InlineCode} from '../../../components/Typography';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/separator"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/separator/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Separator"
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/master/packages/paste-core/components/separator"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/components-separator--horizontal"
+  data={props.data.allPasteComponent.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings} />
+
+<content>
+
+## Guidelines
+
+### About Separator
+
+<Paragraph>{props.pageContext.frontmatter.description}</Paragraph>
+
+It can be used to create separation between large sections of page content, or between smaller parts of single UI controls such as menu items in a menu.
+
+## Examples
+
+### Vertical separation
+
+Vertical separation is achieved by setting the `orientation` property to horizontal. A `verticalSpacing` styling property is provided to enable you to create space above and below the separator. `verticalSpacing` will take any [Spacing Token](/tokens#spacings) as it's value.
```

Comment:
"horizontal" --> "`horizontal`" Or maybe just "by setting `orientation="horizontal"`"

Reply:
"enable you to create" --> "let you create"

Reply:
"it's" --> "its"